sdk/servicebus/service-bus/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Adds new get/set property maxMessageSizeInKilobytes to QueueProperties, TopicProperties, CreateQueueOptions, and CreateTopicOptions. Only applicable for premium namespaces. Added in [#17953](https://github.com/Azure/azure-sdk-for-js/pull/17953)
```

Comment:
This is a preview future, so you may want to update the version number

Reply:
will do.